app/scripts/pages/GenericIconGenerator.js

```diff
@@ -68,6 +68,17 @@ export class GenericIconGenerator extends BaseGenerator {
           defaultValue: 'rgba(0, 0, 0, 0.54)',
           alpha: true
         }),
+		new studio.EnumField('subfolder',{
+			id: 'drawable', title: 'Subfolder', defaultValue: 'drawable',
+			options: [
+				{
+					id: 'drawable', 'title': 'Drawable'
+				},
+				{
+					id: 'mipmap', 'title': 'Mipmap'
+				}
+			]
+		}),
```

Comment:
This new section is indented with tabs. Please use spaces only (like the rest of the file).